tests/src/test/scala/cats/tests/WriterTTests.scala

```diff
@@ -268,4 +268,16 @@ class WriterTTests extends CatsSuite {
 
     Semigroup[WriterT[Id, Int, Int]]
   }
+
+  {
+    // F has a Functor
+    implicit val F: Functor[ListWrapper] = ListWrapper.functor
+
+    CoflatMap[WriterT[ListWrapper, Int, ?]]
+    checkAll("WriterT[Listwrapper, Int, ?]", CoflatMapTests[WriterT[ListWrapper, Int, ?]].coflatMap[Int, Int, Int])
+    checkAll("WriterT[ListWrapper, Int, ?]", SerializableTests.serializable(CoflatMap[WriterT[ListWrapper, Int, ?]]))
+
+    // Id has a Functor
+    CoflatMap[WriterT[Id, Int, ?]]
```

Comment:
Could you also summon the `Functor[WriterT[Id, Int, ?]]` and ``Functor[WriterT[ListWrapper, Int, ?]]` to ensure there aren't ambiguity issues with the `Functor` instances?